frontend/packages/pipelines-plugin/src/components/quicksearch/pipeline-quicksearch-utils.ts

```diff
@@ -0,0 +1,142 @@
+import i18n from 'i18next';
+import { safeLoad } from 'js-yaml';
+import * as _ from 'lodash';
+import { CatalogItem } from '@console/dynamic-plugin-sdk';
+import { coFetch } from '@console/internal/co-fetch';
+import { k8sCreate, k8sUpdate } from '@console/internal/module/k8s';
+import { ClusterTaskModel, TaskModel } from '../../models';
+import { TektonTaskAnnotation } from '../pipelines/const';
+import { CTALabel, TEKTONHUB } from './const';
+
+export const isSelectedVersionInstalled = (item: CatalogItem, selectedVersion: string): boolean => {
+  return item.attributes?.installed === selectedVersion;
+};
+
+export const isTaskVersionInstalled = (item: CatalogItem): boolean => {
+  if (item.attributes?.installed) {
+    return item.attributes.installed !== '';
+  }
+  return false;
+};
```

Comment:
nit: can we have it like this?

```suggestion
export const isTaskVersionInstalled = (item: CatalogItem): boolean =>  !!item.attributes?.installed;
```